### PR TITLE
FIX #80 check $gridField exists before calling getConfig()

### LINKED ISSUE #80: Erroring with elemental-userforms
Apologies in advance, I don't have the specific scenario that caused the error, but I do have the simple fix! While doing SS5 upgrades and reviews, I'm PR'ing back what I can as I go.

Basically `updateCMSFields` in `UserFormUseQueuedExportExtension.php` is expecting `$gridField = $fields->fieldByName('Root.Submissions.Submissions');` to always return a valid gridfield.

When I use this module along with `elemental-userforms`, in some scenario (which I can't recall - but I think is as commonplace as on creation of the userforms element) the `Root.Submissions.Submissions` gridfield doesn't exist, so when `$gridField->getConfig()` is called on null, an error is thrown.

I'm going to submit a PR, ideally could we just add `if (empty($gridField)) return;` between the two lines?

I can do more homework here, I'm just hoping that it's enough of a commonsense edit that you don't need me to 😅 

### PRs
- https://github.com/silverstripe/silverstripe-gridfieldqueuedexport/pull/81

### PR DESCRIPTION
**Pull request by dizzystuff**
Fixes #80

### CODE REVIEW

File: src/Extensions/UserFormUseQueuedExportExtension.php [hunk 1]
```diff
@@ -15,6 +15,7 @@ class UserFormUseQueuedExportExtension extends DataExtension
     public function updateCMSFields(FieldList $fields)
     {
         $gridField = $fields->fieldByName('Root.Submissions.Submissions');
+        if (empty($gridField)) return;
```
Comment (emteknetnz) [inline src/Extensions/UserFormUseQueuedExportExtension.php]:
```suggestion
        if (empty($gridField)) {
            return;
        }
```

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Needs the brackets to pass linting in CI

**Review by GuySartorelli — APPROVED** [review]
Seems sensible. Thanks.

**Comment by dizzystuff** [comment]
Thanks for merging this guys. Would you mind creating a tag for it when you get a moment?

**Comment by GuySartorelli** [comment]
This was merged into the next minor branch, which means it will be included in the next minor release of this module. We don't currently have a date for when that will be.

If this is urgent for you, feel free to create a new PR targetting the `2.8` branch (if this affects that release - target `3.0` otherwise) and @ me and I'll merge and tag that for you.

**Pull request merged**